tests/molecule/resources/playbooks/converge.yml

```
@@ -23,13 +23,12 @@
     community.general.docker_container:
       name: debian
       image: debian
+      env:
+        HOST_ETC: /host/etc
       volumes:
-        - /etc/os-release:/etc/os-release:ro
-        - /etc/redhat-release:/etc/redhat-release:ro
-        - /etc/apt:/etc/apt:ro
-        - /etc/zypp/repos.d:/etc/zypp/repos.d:ro
-        - /etc/yum.repos.d:/etc/yum.repos.d:ro
-        - /etc/pki:/etc/pki:ro
+        - /etc:/host/etc:ro
```

For now, I would prefer that we only mount the files which we know we require, since I believe that's what we're going to end up needing to do in the helm charts & it would be ideal if our tests replicate the customer setup as closely as possible.
```suggestion
        - /etc/apt:/host/etc/apt:ro
        - /etc/zypp/repos.d:/host/etc/zypp/repos.d:ro
        - /etc/yum.repos.d:/host/etc/yum.repos.d:ro
        - /etc/pki:/host/etc/pki:ro
```

---

That means we need to bind mount every file in https://github.com/DataDog/gopsutil/blob/dd/host/host_linux.go#L251-L316

---

I think we should be okay to rely on the `*-release` files